src/main/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStep/config.jelly

```diff
@@ -87,7 +87,7 @@
             <f:optionalBlock title="${%Use custom workspace volume}" checked="${instance.workspaceVolume != null}" inline="true">
```

Comment:
Probably still incorrect as per https://github.com/jenkinsci/kubernetes-plugin/pull/703#discussion_r376597029.

Reply:
With the Descriptor default changed, that should make the config.jelly correct now.

Reply:
Now that `getWorkspaceVolume()` no longer returns null, I'm not sure if this check is even necessary.

Reply:
I _think_ what ought to happen is that the form should be simplified to not use `f:optionalBlock`. The `f:dropdownDescriptorSelector` on its own, with a `default`, should suffice.